RESTserver/src/main/java/com/softserve/if072/restservice/service/CategoryService.java

```diff
@@ -47,16 +43,24 @@ public Category getById(int id) throws DataNotFoundException {
         if (category != null) {
             return category;
         } else {
-            throw new DataNotFoundException(String.format(categoryNotFound, id));
+            throw new DataNotFoundException(String.format("Category with id %d not found", id));
```

Comment:
Because I would have to create lots of properties with different text and I thought that it's easier to write it as a String. Do I have to change it?